l10n_th_personal_income_tax/models/account_move.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+from ...l10n_th_withholding_tax_cert.models.withholding_tax_cert import (
```

Comment:
```suggestion
from odoo.addons.l10n_th_withholding_tax_cert.models.withholding_tax_cert import (
```
In case, the module is not in the same folder

Reply:
Edited.